website/content/docs/agent/autoauth/index.mdx

```diff
@@ -221,5 +259,87 @@ auto_auth {
       path = "/tmp/file-foo"
     }
   }
+
+  sink {
+    type = "file"
```

Comment:
Would it worth explaining that you can have multiple sinks of the same type in a single stanza? I assume it is since you added it, but maybe it would a good note to add.